jenkins/test/run_tests.py

```diff
@@ -116,11 +138,13 @@ def run_validators():
     """ Run all test validators """
     # First, add the validator direcotry to the python path to allow
     # modules to be loaded by pylint
+    # We also add the jenkins/test directory so that github_helpers can be properly loaded.
     pypath = os.getenv("PYTHONPATH", "")
+    tools_test_path = OPENSHIFT_TOOLS_PATH + "jenkins/test/"
     if pypath != "":
-        os.environ["PYTHONPATH"] = VALIDATOR_PATH + os.pathsep + pypath
+        os.environ["PYTHONPATH"] = VALIDATOR_PATH + os.pathsep + tools_test_path + os.pathsep + pypath
```

Comment:
The test failures seen in this PR are fixed by this change to run_tests.py.

They will not take affect on the PR to stg through.